pkg/openshift/certgen/config.go

```diff
@@ -24,6 +27,53 @@ type Config struct {
 	AzureConfig            AzureConfig
 }
 
+// TemplateInterface created to simplify testing
+type templateInterface interface {
```

Comment:
since none of has ended up being used, might as well remove the changes associated with this from the PR - might want to stash them locally in case they turn out to be useful in the future though